modules/olmv1-clusteroperator.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * operators/operator-reference.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cluster-operators-ref-olmv1_{context}"]
+= {olmv1-first} Operator
+
+[discrete]
+== Purpose
+
+Starting in {product-title} 4.18, {olmv1-first} is enabled by default alongside the {olmv0}. This next-generation iteration provides an updated framework that evolves many of the {olmv0} concepts that enable cluster administrators to extend capabilities for their users.
+
+{olmv1} manages the lifecycle of the new `ClusterExtension` object, which includes Operators via the `registry+v1` bundle format, and controls installation, upgrade, and role-based access control (RBAC) of extensions within a cluster.
+
+In {product-title}, {olmv1} is provided by the `olm` cluster Operator.
+
+[discrete]
+== Components
+
+{olmv1-first} comprises the following component projects:
+
+Operator Controller:: The central component of {olmv1} that extends Kubernetes with an API through which users can install and manage the lifecycle of Operators and extensions. It consumes information from catalogd.
+
+Catalogd:: A Kubernetes extension that unpacks file-based catalog (FBC) content packaged and shipped in container images for consumption by on-cluster clients. As a component of the {olmv1} microservices architecture, catalogd hosts metadata for Kubernetes extensions packaged by the authors of the extensions, and as a result helps users discover installable content.
+
+[discrete]
+== CRDs
+
+* `clusterextension.olm.operatorframework.io`
+** Scope: Cluster
+** CR: `ClusterExtension`
+
+* `clustercatalog.olm.operatorframework.io`
+** Scope: Cluster
+** CR: `ClusterCatalog`
+
+[discrete]
+== Project
+
+* link:https://github.com/operator-framework/operator-controller[operator-framework/operator-controller]
```

Comment:
I'm guessing these links to GitHub have been approved? https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Reply:
Yeah, per the approved layout at the time that the "Cluster Operators reference" assembly was introduced, the support team approved this type of usage.